ui/src/kapacitor/components/config/AlertaConfig.js

```diff
@@ -20,8 +20,10 @@ class AlertaConfig extends Component {
       url: this.url.value,
     }
 
-    this.props.onSave(properties)
-    this.setState({testEnabled: true})
+    const success = await this.props.onSave(properties)
+    if (success) {
+      this.setState({testEnabled: true})
```

Comment:
Would it be better behavior to say
```
const success = await this.props.onSave(properties)
this.setState({testEnabled: success})
```

Would this disable the test button every time you try to save a bad config?

Reply:
I didn't do this because the behavior would be the same, but it is more readable this way don't feel strongly either way.